platform/src/common/components/Layout/index.jsx

```diff
@@ -78,7 +74,7 @@ const Layout = ({ children, topbarTitle, noBorderBottom }) => {
     };
 
     setChartProperties();
-  }, [userInfo, dispatch]);
+  }, [userInfo, userPreferences, dispatch]);
```

Comment:
This chartData related code is in the wrong place. Should be under the analytics page.